index.html

```diff
@@ -59,17 +67,21 @@ <h2 class="title-2">Software Developer</h2>
     </section>
     <img class="vector-img" src="sprites/Vector4.png" alt="End of section 2">
     <section id="section-3">
+        <div>
+            <h2>Work</h2><img src="sprites/Group55.png" alt="Down Arrow">
+        </div>
         <nav>
-            <a href="#"><img src="sprites/Disabled.png" alt="Github icon"></a>
-            <a href="#"><img src="sprites/Disabled-1.png" alt="Linkdin icon"></a>
+            <a href="https://github.com/wandergithub"><img src="sprites/Disabled.png" alt="Github icon"></a>
+            <a href="https://www.linkedin.com/in/wander-a-gonzalez-53127b205/"><img src="sprites/Disabled-1.png" alt="Linkdin icon"></a>
             <a href="#"><img src="sprites/Enabled.png" alt="Angelist icon"></a>  
-            <a href="#"><img src="sprites/Disabled-2.png" alt="Twitter icon"></a>
+            <a href="https://twitter.com/wanderklk1"><img src="sprites/Disabled-2.png" alt="Twitter icon"></a>
             <a href="#"><img src="sprites/Disabled-3.png" alt="Medium icon"></a>
         </nav>
-        <div>
+        <div id="work-section">
             <img src="sprites/Group55.png" alt="Down Arrow"><h2>Work</h2>
         </div>
         <h2 class="title">MY RECENT WORK</h2>
+        <h2 class="title desktop-v">My recent work</h2>
     </section>
     <img class="vector-img" src="sprites/Vector4.png" alt="End of section 3">
     <section id="section-4">
```

Comment:
- [ ]  Your design is looking great in both versions, well done! However, we still need to invert every other card to look like the Figma template:

Template:
![Captura de pantalla 2022-03-24 a la(s) 7 04 24 p  m](https://user-images.githubusercontent.com/83717285/160029965-504e0d7e-44da-4790-8d67-c6fb90c2b2f8.png)

Your design:
![Captura de pantalla 2022-03-24 a la(s) 7 04 42 p  m](https://user-images.githubusercontent.com/83717285/160029991-dbef8a39-e55e-48ab-995e-4d0c1e89bb8c.png)

Reply:
use a  row reverse attribute css